pom.xml

```diff
@@ -394,6 +395,18 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-source-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>attach-sources</id>
+						<goals>
+							<goal>jar</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
I noticed we weren't generating sources since June, although they make it easier to debug in downstream projects. I'm not sure why I have to include this plugin explicitly though, since I think other projects do this by default.